articles/intro-to-katas.md

```diff
@@ -3,22 +3,23 @@ title: Introduction to the Quantum Katas
 description: Learn about the katas (training exercises) provided with the Microsoft Quantum Development Kit (QDK)
 author: natke
 ms.author: nakersha 
-ms.date: 10/17/2019
+ms.date: 06/02/2020
 ms.topic: overview
 uid: microsoft.quantum.overview.katas
 ---
 
 # Learn quantum computing with the Quantum Katas
 
-[The Quantum Katas](https://github.com/Microsoft/QuantumKatas/) is an open-source collection of self-paced tutorials and sets of programming exercises aimed at teaching you elements of quantum computing and Q# programming at the same time.
+[The Quantum Katas](https://github.com/Microsoft/QuantumKatas/) are open-source, self-paced tutorials and programming exercises aimed at teaching the elements of quantum computing and Q# programming at the same time.
 
 ## Learning by doing
 
 The tutorials and exercises collected in this project emphasize learning by doing: they offer programming tasks that cover certain topics which progress from very simple to quite challenging. Each task asks you to fill in some code; the first tasks might require just one line, and the later ones might require a sizable fragment of code.
 
-Most importantly, the katas include testing frameworks that sets up, runs and validates the solutions to the tasks. This allows you to get immediate feedback on your solution and to reconsider your approach if it is incorrect.
+Most importantly, the katas include testing frameworks that allow you to
```

Comment:
I feel that the important distinction is lost here: the testing framework doesn't allow the learner to do set up and validation, it does that automatically so that the learner just gets the feedback. Figuring out a way to test the solution is quite tricky, especially for a beginner, so it's nice to emphasize that one doesn't need to do it with the Katas.

Reply:
Thanks!